src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -28,22 +25,38 @@ partial class ApplicationEngine
         /// </summary>
         public static readonly InteropDescriptor System_Contract_CreateStandardAccount = Register("System.Contract.CreateStandardAccount", nameof(CreateStandardAccount), 0_00010000, CallFlags.None, true);
 
-        protected internal void CreateContract(byte[] script, byte[] manifest)
+        private UInt160 ContractHash(NefFile nef)
```

Comment:
```suggestion
        public static UInt160 ContractHash(UInt160 sender, NefFile nef)
```

Move to Helper.cs? Then neo-node/modules can call it.

Reply:
Done